src/sql/parts/dashboard/dashboard.module.ts

```diff
@@ -137,7 +138,7 @@ export class DashboardModule {
 	constructor(
 		@Inject(forwardRef(() => ComponentFactoryResolver)) private _resolver: ComponentFactoryResolver,
 		@Inject(BOOTSTRAP_SERVICE_ID) private _bootstrapService: IBootstrapService,
-		@Inject(forwardRef(() => DashboardServiceInterface)) private _bootstrap: DashboardServiceInterface,
+		@Inject(forwardRef(() => CommonServiceInterface)) private _bootstrap: DashboardServiceInterface,
```

Comment:
Per previous comments, these should be consistent. In this case, likely use DashboardServiceInterface for the DashboardModule since it'll need some of its features when bootstrapping.